assertk-jvm/src/test/kotlin/test/assertk/AssertBlockSpec.kt

```diff
@@ -5,7 +5,7 @@ import assertk.assertions.isEqualTo
 import assertk.assertions.isNegative
 import assertk.assertions.isPositive
 import assertk.assertions.message
-import org.junit.jupiter.api.Test
+import org.junit.Test
```

Comment:
These should be kotlin test annotation right?

Reply:
Can be. But when working on the java side, IntelliJ adds the junit annotation. But it's easy to exclude. Both are fine to me. Any preference ?